src/components/ProjectSummary/ProjectSummary.styled.ts

```diff
@@ -0,0 +1,101 @@
+import { css, styled } from 'styled-components';
+
+const BasicColumnStyle = css`
```

Comment:
jak piszesz same css to zmienna z małej litery

Reply:
done